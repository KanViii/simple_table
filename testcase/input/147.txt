INSERT x string
BEGIN
INSERT x number
INSERT y string
ASSIGN x 5
ASSIGN y '5'
BEGIN
INSERT z number
INSERT abc string
INSERT cba number
PRINT
RPRINT
END
RPRINT
LOOKUP x
LOOKUP y
END
PRINT