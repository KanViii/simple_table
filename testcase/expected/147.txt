success
success
success
success
success
success
success
success
x//1 y//1 z//2 abc//2 cba//2
cba//2 abc//2 z//2 y//1 x//1
y//1 x//1
1
1
x//0